templates/Web/Projects/ReactDefault/src/components/NavBar/index.jsx

```diff
@@ -7,12 +8,12 @@ export default function NavBar() {
   return (
     <React.Fragment>
       <div className={styles.skipLink}>
-        <a href="#mainContent">Skip to Main Content</a>
+        <Link to="#mainContent">Skip to Main Content</Link>
```

Comment:
I think that this line should remain as: 
```html
<a href="#mainContent">Skip to Main Content</a>
```

I feel like the Link here will break the skip link. Thoughts?

Reply:
Links do end up rendering an anchor tag in the DOM, but good point - i'm not sure how it treats skip links so i'll revert it.